pkg/cloudprovider/packing/nodecapacity.go

```diff
@@ -17,29 +17,29 @@ package packing
 import (
 	"fmt"
 
-	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/awslabs/karpenter/pkg/cloudprovider"
```

Comment:
consider nesting this packing library under /pkg

Reply:
👍  done!